pkg/git/gitlab/gitlab.go

```diff
@@ -230,23 +230,27 @@ func (g *Gitlab) FullURL() *url.URL {
 // it will return false.
 func (g *Gitlab) IsType(URL *url.URL) (bool, error) {
 
+	if strings.Contains(URL.Host, "gitlab.com") {
+		return true, nil
+	}
+
 	httpClient := &http.Client{
 		Timeout: 10 * time.Second,
 	}
 
-	gitlabURL := URL.Scheme + "://" + URL.Host + "/users/sign_in'"
+	gitlabURL := URL.Scheme + "://" + URL.Host + "/users/sign_in"
 
 	resp, err := httpClient.Get(gitlabURL)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("can't query gitlab login page: %v", err)
```

Comment:
```suggestion
		return false, fmt.Errorf("can't query gitlab login page: %w", err)
```

Reply:
What's the reason to stick with `%v` here?

Reply:
Habit. Changed it to `%w`.